packages/core/src/common/message-service.ts

```diff
@@ -195,12 +195,13 @@ export class MessageService {
         const report = (update: ProgressUpdate) => {
             this.client.reportProgress(id, update, message, cancellationSource.token);
         };
+        const type = message.type ? message.type : MessageType.Progress;
```

Comment:
Nitpick:
```suggestion
        const type = message.type ?? MessageType.Progress;
```

Reply:
Thanks for the fast feedback and your suggestion!
I applied this change and also used the ?? operator in `notification-component.tsx` with the newly force-pushed commit 9f7c090.